source/ProSymbolEditor/Models/MilitaryOverlayDataModel.cs

```diff
@@ -33,60 +33,84 @@ public class MilitaryOverlayDataModel
         private bool _schemaExists;
         private string _databaseName;
 
+        private string _egdbConnectionString;
+
         public MilitaryOverlayDataModel()
         {
             _schemaExists = false;
         }
 
-        Dictionary<string, bool> GetFeatureClassExistsMap(ProSymbolUtilities.SupportedStandardsType standard)
+        Dictionary<string, bool> GetFeatureClassExistsMap(ProSymbolUtilities.SupportedStandardsType standard,
+            Geodatabase gdb = null)
         {
+
+            string prefixName = string.Empty;
+            if (gdb != null)
+            {
+                GeodatabaseType gdbType = gdb.GetGeodatabaseType();
+                if (gdbType == GeodatabaseType.RemoteDatabase)
+                {
+                    // if an SDE/EGDB, then feature class name format will differ:
+                    // Database + User + Feature Class Name 
+                    ConnectionProperties cps = gdb.GetConnectionProperties();
+                    prefixName = cps.Database + "." + cps.User + ".";
+
+                    // Also save this connection string to identify this EGDB later 
+                    _egdbConnectionString = (gdb as Datastore).GetConnectionString();
+                }
+                else
+                {
+                    _egdbConnectionString = string.Empty;
+                }
+            }
+
             Dictionary<string, bool> featureClassExists = new Dictionary<string, bool>();
 
             if (standard == ProSymbolUtilities.SupportedStandardsType.mil2525c_b2)
             {
                 // 2525c_b2
-                featureClassExists.Add("Activities", false);
-                featureClassExists.Add("Air", false);
-                featureClassExists.Add("ControlMeasuresAreas", false);
-                featureClassExists.Add("ControlMeasuresLines", false);
-                featureClassExists.Add("ControlMeasuresPoints", false);
-                featureClassExists.Add("Installations", false);
-                featureClassExists.Add("LandEquipment", false);
-                featureClassExists.Add("METOCAreas", false);
-                featureClassExists.Add("METOCLines", false);
-                featureClassExists.Add("METOCPoints", false);
-                featureClassExists.Add("SeaSubsurface", false);
-                featureClassExists.Add("SeaSurface", false);
-                featureClassExists.Add("SIGINT", false);
-                featureClassExists.Add("Space", false);
-                featureClassExists.Add("Units", false);
+                featureClassExists.Add(prefixName + "Activities", false);
```

Comment:
I didn't really think about, but this class could indeed use the existing list of these from SymbolSetMappings class - the project would just need refactored some to move the object hierarchy/containers a bit - just let me know if you want me to do as part of this PR